From 0653b39a0afab570bc4a095a33dd3b43d6504877 Mon Sep 17 00:00:00 2001
From: John Williams <porschemad911@gmail.com>
Date: Sun, 23 Aug 2026 01:10:00 +1000
Subject: [PATCH 42/42] power: supply: rk818-charger: Fix values for
 POWER_SUPPLY_PROP_CONSTANT_CHARGE_VOLTAGE_MAX and
 POWER_SUPPLY_PROP_ENERGY_FULL_DESIGN

Align with devicetree
---
 drivers/power/supply/rk818_charger.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/power/supply/rk818_charger.c b/drivers/power/supply/rk818_charger.c
index d8c3b7a09d76..0a93440c0c03 100644
--- a/drivers/power/supply/rk818_charger.c
+++ b/drivers/power/supply/rk818_charger.c
@@ -509,7 +509,7 @@ static int rk818_charger_get_property(struct power_supply *psy,
                 return rk818_charger_get_current_max(cg, &val->intval);
 
 	case POWER_SUPPLY_PROP_CONSTANT_CHARGE_VOLTAGE_MAX:
-		val->intval = 4350000;
+		val->intval = 4200000;
 		break;
 
 	case POWER_SUPPLY_PROP_CONSTANT_CHARGE_CURRENT_MAX:
@@ -517,7 +517,7 @@ static int rk818_charger_get_property(struct power_supply *psy,
 		break;
 
 	case POWER_SUPPLY_PROP_ENERGY_FULL_DESIGN:
-		val->intval = 11400000;
+		val->intval = 14800000;
 		return 0;
 
 	case POWER_SUPPLY_PROP_ENERGY_EMPTY_DESIGN:
-- 
2.47.3

